add Alluxio example

saveveltri opened the pull request.

saveveltri committed:
add Alluxio example

saveveltri committed:
Merge commit '6acc857f7b0a842c166c6a9a84d3479c155c0f84' into RBFDD-432

saveveltri committed:
move AlluxioAPIExample

saveveltri committed:
add rap repository reference

saveveltri committed:
alignment fixed

stefanobaghino commented on snippets/include-alluxio.xml:
```diff
@@ -0,0 +1,5 @@
+<dependency>
```
Would it be feasible to refer directly to the `pom.xml`?

stefanobaghino commented on snippets/rbp-repository.xml:
```diff
@@ -0,0 +1,22 @@
+<repositories>
```
Would it be feasible to refer directly to the `pom.xml`?

saveveltri commented:
Thanks Stefano for the review. One point. Addressing your comments would remove redundant files, but, on the other hand, it means that every updates on the source file (the pom.xml in this case) can affect all the doc files it is referred. It could get tough to handle this.
What is your thought?

Thanks

stefanobaghino commented:
@saveveltri The safest way to use the `pom.xml` without affecting other pages is to append your edits at the end, as long as you don't push other lines down there's no risk. I did the same for [this commit][1]. We can check how it turns out this way tomorrow, if you think the change is feasible. 😉 

[1]: https://github.com/radicalbit/rbp-docs-code/pull/5/files#diff-02bea068fb725286d736b044a68d9d17L114

saveveltri committed:
add isolated project for Alluxio API

saveveltri commented:
An ad hoc project for Alluxio API has been created based on our morning thoughts

stefanobaghino commented:
Very good, is it safe to remove code under `mvn` now? Or is it still referenced in the documentation?

saveveltri commented:
if no one else is currently use code under `mvn` I will gladly remove it, otherwise we could arrange a `clean up` PR @alkagin @maocorte @giampaolotrapasso

alkagin commented:
@saveveltri I've already open a PR to move the connectors examples from mvn folder to independent projects, but I still prefer a clean up PR to avoid a huge rebase in the opened PR.

stefanobaghino commented:
@saveveltri I think you can safely remove the duplicate examples you introduced with this PR under `mvn` without affecting others' work.

giampaolotrapasso commented:
Thanks
